src/hotspot/share/jvmci/vmStructs_jvmci.cpp

```diff
@@ -180,7 +180,7 @@
   nonstatic_field(JavaThread,                  _jvmci_counters,                               jlong*)                                \
   nonstatic_field(JavaThread,                  _should_post_on_exceptions_flag,               int)                                   \
   nonstatic_field(JavaThread,                  _jni_environment,                              JNIEnv)                                \
-  nonstatic_field(JavaThread,                  _reserved_stack_activation,                    address)                               \
+  nonstatic_field(JavaThread,                  _stack_overflow_state._reserved_stack_activation, address)                            \
```

Comment:
This doesn't look right. I thought these had to be direct entries for fields in given classes - not indirections??

Reply:
Yes, this does work.  I confirmed it on the graal internal slack page with Doug Simon.